source/io/module_eggShell.js

```diff
@@ -705,7 +705,8 @@ var assignEggShell;
                 origPrintErr(textErr);
             };
 
-            var result = Module._EggShell_REPL(Module.eggShell.v_userShell, viaTextPointer, viaTextLength);
+            var debugging = (config === null || config === undefined || typeof config.debugging !== 'boolean') ? false : config.debugging;
```

Comment:
A more common pattern if we can't use optional chaining would be to check if config is truthy and then check the property. ie
```suggestion
            var debugging = (config && typeof config.debugging === 'boolean') ? config.debugging : false;
```